analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/DeliveringDebugFeaturesInProductionBase.cs

```diff
@@ -31,7 +31,8 @@ public abstract class DeliveringDebugFeaturesInProductionBase<TSyntaxKind> : Tra
         where TSyntaxKind : struct
     {
         protected const string DiagnosticId = "S4507";
-        protected const string MessageFormat = "Make sure this debug feature is deactivated before delivering the code in production.";
+        protected const string StartupDevelopment = "StartupDevelopment";
+        private const string MessageFormat = "Make sure this debug feature is deactivated before delivering the code in production.";
```

Comment:
There's stylecope formatting issue on line 40 with `);` that can be easily fixed (clean as you code)